diff --git a/node_modules/vuepress-theme-reco/components/Footer.vue b/node_modules/vuepress-theme-reco/components/Footer.vue
index 76b93f7..8074ef3 100644
--- a/node_modules/vuepress-theme-reco/components/Footer.vue
+++ b/node_modules/vuepress-theme-reco/components/Footer.vue
@@ -21,6 +21,10 @@
       <reco-icon icon="reco-eye" />
       <AccessNumber idVal="/" />
     </span>
+    <!-- 尾部修改 -->
+    <span>
+      <a href="https://creativecommons.org/licenses/by-nc/4.0/deed.zh">本博客采用“保持署名—非商用”创意共享4.0许可证</a>
+    </span>
     <p class="cyber-security" v-if="$themeConfig.cyberSecurityRecord">
       <img src="https://img.alicdn.com/tfs/TB1..50QpXXXXX7XpXXXXXXXXXX-40-40.png" alt="">
       <a :href="$themeConfig.cyberSecurityLink || '#'">{{ $themeConfig.cyberSecurityRecord }}</a>
diff --git a/node_modules/vuepress-theme-reco/components/HomeBlog/index.vue b/node_modules/vuepress-theme-reco/components/HomeBlog/index.vue
index 4c2e13b..77eca9e 100644
--- a/node_modules/vuepress-theme-reco/components/HomeBlog/index.vue
+++ b/node_modules/vuepress-theme-reco/components/HomeBlog/index.vue
@@ -84,7 +84,8 @@ export default defineComponent({
     const bgImageStyle = computed(() => {
       const url = instance.$frontmatter.bgImage
         ? instance.$withBase(instance.$frontmatter.bgImage)
-        : require('../../images/bg.svg')
+        // : require('../../images/bg.svg')
+        : require('/img/modao.jpg')
 
       const initBgImageStyle = {
         textAlign: 'center',
diff --git a/node_modules/vuepress-theme-reco/styles/code.styl b/node_modules/vuepress-theme-reco/styles/code.styl
index b8551c3..dea4f23 100644
--- a/node_modules/vuepress-theme-reco/styles/code.styl
+++ b/node_modules/vuepress-theme-reco/styles/code.styl
@@ -1,6 +1,7 @@
 .content__default
  code
-   color lighten($textColor, 20%)
+   //color lighten($textColor, 20%)
+   color red
    padding 0.25rem 0.5rem
    margin 0
    font-size 0.85em
